PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,39 @@
-## Description of Work
-<!--add description of work here 🙏🏼 -->
+## Description
+<!--add a short description of work here 🙏🏼 -->
 
 ### Change List
+<!-- More detailed list of the major changes -->
+- item 1
+- item 2
+
+<details>
+<summary>Extra Stuff</summary>
+  <!-- select all that apply -->
+  <!-- More details from GitHub -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting -->
+
+  ### Added/updated tests?  
+  - [ ] Yes
+  - [ ] No, and this is why: _please replace this line with details on why tests
+        have not been included_
+        
+  ### Diagrams
```

Comment:
`Mint` has a `/docs/diagrams` folder. Would it be better to have diagrams as a part of the PR changes instead of just the description?

Reply:
I guess my intention here was more for "un-official" diagrams that help convey the changes in the PR better than words.